Modules/ManualAssessment/classes/Members/class.ilManualAssessmentMembersStorageDB.php

```diff
@@ -0,0 +1,118 @@
+<?php
+require_once 'Modules/ManualAssessment/interfaces/Members/interface.ilManualAssessmentMembersStorage.php';
+require_once 'Modules/ManualAssessment/classes/Members/class.ilManualAssessmentMembers.php';
+require_once 'Modules/ManualAssessment/classes/Members/class.ilManualAssessmentMember.php';
+require_once 'Modules/ManualAssessment/classes/class.ilObjManualAssessment.php';
+/**
+ * Store member infos to DB
+ * @inheritdoc
+ */
+class ilManualAssessmentMembersStorageDB implements ilManualAssessmentMembersStorage {
+
+	protected $db;
+
+	public function __construct($ilDB) {
+		$this->db = $ilDB;
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function loadMembers(ilObjManualAssessment $obj) {
+		$members = new ilManualAssessmentMembers($obj);
+		$obj_id = $obj->getId();
+		$sql = $this->loadMembersQuery($obj_id);
+		$res = $this->db->query($sql);
+		while($rec = $this->db->fetchAssoc($res)) {
+			$members = $members->withAdditionalRecord($rec);
+		}
+		return $members;
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function loadMember(ilObjManualAssessment $obj, ilObjUser $usr) {
+		$obj_id = $obj->getId();
+		$usr_id = $usr->getId();
+		$sql = 'SELECT massme.*'
```

Comment:
* in einem SELECT ist eigentlich ziemlich böse. Sind es zu viele Spalten um hier jede auf zu führen?